src/grpc_client.cc

```diff
@@ -192,6 +193,22 @@ int32_t CloseClient(grpc_labview::LabVIEWgRPCClient* client)
     return 0;
 }
 
+// Signal a lv occurence for a client call from async c++ thread
+void SignalOccurenceForClientCall(grpc_labview::ClientCall* clientCall)
```

Comment:
Should we name this method `CheckActiveAndSignalOccurence` to signify that we will fire the occurrence only if the client call is active?